template/build.ps1

```diff
@@ -0,0 +1,39 @@
+cd $PSScriptRoot
+
+$repoFolder = $PSScriptRoot
+$env:REPO_FOLDER = $repoFolder
+
+$koreBuildZip="https://github.com/aspnet/KoreBuild/archive/dev.zip"
+if ($env:KOREBUILD_ZIP)
+{
+    $koreBuildZip=$env:KOREBUILD_ZIP
+}
+
+$buildFolder = ".build"
+$buildFile="$buildFolder\KoreBuild.ps1"
+
+if (!(Test-Path $buildFolder)) {
+    Write-Host "Downloading KoreBuild"    
+    $tempFolder=$env:TEMP + "\KoreBuild-" + [guid]::NewGuid()
```

Comment:
Suggest `[System.IO.Path]::GetRandomFileName()` instead of `[guid]::NewGuid()`. Shorter and still unique enough.